targeted-build-script.sh

```diff
@@ -42,23 +60,22 @@ for repo in $repos; do
   num_tags=$(echo ${repo} | awk -F_ '{print $1}')
 
   if [[ $num_tags -gt 0 ]]; then
-
+    count=0
+    echo $count
+    echo $max_concurrent
```

Comment:
You might want to try using parallel instead. Would save you from for loop cycling. Not sure if it would do much for speed but could be worth a shot